coral/src/domain/connector/index.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  approveConnectorRequest,
+  declineConnectorRequest,
+  getConnectorRequests,
+  getConnectorRequestsForApprover,
+  deleteConnectorRequest,
+} from "src/domain/connector/connector-api";
+import {
+  ConnectorRequest,
+  ConnectorRequestsForApprover,
+} from "src/domain/connector/connector-types";
+
+export type { ConnectorRequestsForApprover, ConnectorRequest };
```

Comment:
Do we need `ConnectorRequest` on the "outside"? 🤔

Reply:
I would expect for the Details modals when approving, as a representation of a single entity?

Reply:
💡  yes, you're absolutely right.